src/libprojectM/Common.hpp

```diff
@@ -30,6 +30,7 @@
 #include <typeinfo>
 #include <cstdarg>
 #include <cassert>
+#include "config.h"
```

Comment:
in re https://github.com/projectM-visualizer/projectm/pull/53#discussion_r180357094 -
is this necessary in Common.hpp? this header is (probably) needed to be "public" and shouldn't include config.h which is "private" to the project

Reply:
It could be deleted, I should have defined "USE_FTGL" with a cflag instead.